lib/src/components/Radio/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ForwardRefRenderFunction } from "react";
+
+import { TRadioGroupProps } from "../radio.types";
+import { Flex } from "src/components/Flex";
+import { assignInlineVars } from "@vanilla-extract/dynamic";
+import {
+  radioGroupChildClass,
+  radioGroupSpacing,
+  radioGroupThemeClass,
+  radioGroupThemeVars
+} from "../radio.styles.css";
+import { eliminateUndefinedKeys } from "src/utils/object-utils";
+
+const RadioGroup: ForwardRefRenderFunction<HTMLDivElement, TRadioGroupProps> = (
+  {
+    children,
+    className,
+    style,
+    orientation = "horizontal",
+    Spacing = "xs",
+    ...nativeProps
+  },
+  ref
+) => {
+  const assignVariables = assignInlineVars(
+    eliminateUndefinedKeys({
+      [radioGroupThemeVars.radioGroupStyleSpacing]: radioGroupSpacing[Spacing]
+        ? radioGroupSpacing[Spacing]
+        : undefined
```

Comment:
why not use the `gap` prop of `Flex` here?

Reply:
Done